client/battle/CBattleInterfaceClasses.h

```diff
@@ -51,9 +52,10 @@ class CBattleConsole : public CIntObject
 class CBattleHero : public CIntObject
 {
 	void switchToNextPhase();
+
 public:
 	bool flip; //false if it's attacking hero, true otherwise
-	CDefHandler *dh, *flag; //animation and flag
+	CDefHandler * dh, * flag; //animation and flag

```

Comment:
@ArseniyShestakov so maybe we should add some rules to coding guidelines about multi elements intantiation with comma?